index.html

```diff
@@ -6,5 +6,6 @@
     <title>Welcome to Potionfy</title>
 </head>
 <body>
+    <h1>Welcome to Potionfy! Enjoy yout stay</h1>
```

Comment:
Could you remove the 'Enjoy you stay' string and make the 'Welcome to Portionfy!' string red?